### PR TITLE
started working on the project

### PR DESCRIPTION
**Pull request by sinjkeee**
added a request to the server and getting/parse a json file

### CODE REVIEW

File: WeatherApp/Supporting files/Constants.swift [hunk 1]
```diff
@@ -0,0 +1,3 @@
+import Foundation
+
+let apiKey = "f5b1b015006e1802f35638d295100520"
```
Comment (vkhadyka) [inline WeatherApp/Supporting files/Constants.swift:3]: Is that good way to store API key as a constant?

File: WeatherApp/View Controllers/ViewController.swift [hunk 2]
```diff
@@ -0,0 +1,13 @@
+import UIKit
+
+class ViewController: UIViewController {
```
Comment (vkhadyka) [inline WeatherApp/View Controllers/ViewController.swift]: I think we can rename our `ViewController` class for better experience

File: WeatherApp/View Controllers/ViewController.swift [hunk 3]
```diff
@@ -1,13 +1,13 @@
 import UIKit
 
-class ViewController: UIViewController {
+class MainViewController: UIViewController {
```
Comment (vkhadyka) [inline WeatherApp/View Controllers/ViewController.swift]: you forgot rename physical name of file `WeatherApp/View Controllers/ViewController.swift`

**Review by vkhadyka — COMMENTED** [review]
is `NetworkWeatherManager.swift` is model? why manager locate in `Models` directory?

File: WeatherApp/View Controllers/ViewController.swift (hunk 3 above)
Comment (sinjkeee) [reply WeatherApp/View Controllers/ViewController.swift]: damn right

**Pull request merged**